homeassistant/components/nam/air_quality.py

```diff
@@ -0,0 +1,90 @@
+"""Support for the Nettigo Air Monitor air_quality service."""
+from __future__ import annotations
+
+from homeassistant.components.air_quality import AirQualityEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import NAMDataUpdateCoordinator
+from .const import AIR_QUALITY_SENSORS, DEFAULT_NAME, DOMAIN
+
+PARALLEL_UPDATES = 1
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Add a Nettigo Air Monitor entities from a config_entry."""
+    coordinator = hass.data[DOMAIN][entry.entry_id]
+
+    entities = []
+    for sensor in AIR_QUALITY_SENSORS:
+        if f"{sensor}_p1" in coordinator.data:
```

Comment:
Where did the _p1 come from, I suspect you set it somewhere else. If so it should be a const.

Reply:
The device sends data with this suffix. You're right, I'll make it a constant.

Reply:
👍 seems it is still TBD